tests/network/vmi_istio.go

```diff
@@ -132,7 +132,7 @@ var istioTests = func(vmType VmType) {
 			virtClient, err = kubecli.GetKubevirtClient()
```

Comment:
Do you want to remove this ?
all over if so please as well

Reply:
It is used on line 139.

Reply:
Right, can it move closer to 139 please ?

Reply:
I prefer not to touch it at all.